dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -569,6 +572,15 @@ public AuthenticationProvider getUserAuthProvider() {
         return userAuthProvider;
     }
 
+    public String getUserLocalizedNotificationsLanguageForDisplay() {
+        String displayLanguage = StringUtils.EMPTY;
+        if(isUserLanguageConfigured()) {
```

Comment:
Do we really want to distinguish this situation. Why don't just base on what Locale object returns?